contrib/datawave-quickstart/bin/services/accumulo/bootstrap.sh

```diff
@@ -109,17 +109,15 @@ DW_ZOOKEEPER_CMD_FIND_ALL_PIDS="ps -ef | grep 'zookeeper.server.quorum.QuorumPee
 DW_ACCUMULO_CMD_START="( cd ${ACCUMULO_HOME}/bin && ./accumulo-cluster start )"
 DW_ACCUMULO_CMD_STOP="( cd ${ACCUMULO_HOME}/bin && ./accumulo-cluster stop )"
 DW_ACCUMULO_CMD_FIND_ALL_PIDS="pgrep -d ' ' -f 'o.start.Main manager|o.start.Main tserver|o.start.Main monitor|o.start.Main gc|o.start.Main tracer'"
+DW_ACCUMULO_CMD_FIND_ALL_PIDS_COUNT="pgrep -f 'o.start.Main manager|o.start.Main tserver|o.start.Main monitor|o.start.Main gc' | wc -l"
```

Comment:
If the goal is to allow start commands to be retried on a partially-started Accumulo or Hadoop, then I think I see the benefit of the pid counting logic, since that could definitely save the user some time.

But rather than duplicating the ```pgrep -f '...'``` command just to determine the count via ```wc -l```, it would be better just to leverage the existing ```*PidList``` functions, so that we don't incur the extra maintenance cost of keeping both commands in sync over time.

For example, something like this could suffice instead...
```
...
# Convert pid list string into local array
local pidArray=( $(accumuloPidList) )

[[ ${#pidArray[@]} -eq 4 ]] && return 0 || return 1
...
```